react-time-track/src/components/list-projects.js

```diff
@@ -39,6 +39,8 @@ function ListProjects() {
                 <span>{project.name}</span>
               </Link>
               <Circle styles={calculateRisk(project.weekly)}>
+                {console.log(project.real_cost)}
```

Comment:
Are these couple of lines required.
Don't mind this message if you have future plans for them.